starter-code/css/main.css

```diff
@@ -6,3 +6,182 @@ Green: #00B172
 White: #FFF
 
 */
+
+/* ---- reset ---- */
+
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+ul, li{
+  margin: 0;
+  padding: 0;
+}
+li{
+  list-style: none;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+
+/* ---- typography ---- */
+
+body {
+  color: #111;
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: 16px;
+  line-height: 147%;
+}
+
+h1 {
+  font-size: 4rem;
+  line-height: 5rem;
+  text-align: center;
+  font-weight: 400;
+}
+
+h2{
+  text-align: center;
+  font-size: 1.75rem;
+  line-height: 2.5rem;
+  font-weight: 100;
+}
+
```

Comment:
Si mantenéis un espacio entre las rules hacedlo siempre, queda un código más limpio.

Reply:
👍🏼